RELEASING.md

```diff
@@ -2,6 +2,12 @@
 
 This documents describes the manual steps required to publish a release to maven central.
 
+## Release cadence
+
+This repository targets monthly minor releases from the `main` branch roughly a week after
+the monthly minor release
+of [opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java)).
```

Comment:
I think we want to follow the instrumentation repo, because we leverage the bom from `opentelemetry-java-instrumentation` which, in turn, versions the core/sdk repo deps.
```suggestion
of [opentelemetry-java-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation)).
```